Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+  </data>
+  <data name="SearchASUPTBS.PlaceholderText" xml:space="preserve">
+    <value>Busca cualquier cosa; subreddits, usuarios, post, etc.</value>
+  </data>
+  <data name="SearchForTB.Text" xml:space="preserve">
+    <value>Buscar:</value>
+  </data>
+  <data name="SearchTBs.Text" xml:space="preserve">
+    <value>Buscar</value>
+  </data>
+  <data name="searchTextBlock.Text" xml:space="preserve">
+    <value>Buscar</value>
+  </data>
+  <data name="SendingCommentTB.Text" xml:space="preserve">
+    <value>Enviando comentario...</value>
+  </data>
+  <data name="settingsGridTB.Text" xml:space="preserve">
+    <value>Configuración</value>
+  </data>
+  <data name="ShareHeartTB.Text" xml:space="preserve">
+    <value>Comparte el ❤</value>
+  </data>
+  <data name="ShowMoreTB.Text" xml:space="preserve">
+    <value>Mostrar más</value>
+  </data>
+  <data name="signInGridTB.Text" xml:space="preserve">
+    <value>Entrar</value>
+  </data>
+  <data name="subscribersTextBlock.Text" xml:space="preserve">
+    <value>73,000 subscriptores</value>
+  </data>
+  <data name="textBlock.Text" xml:space="preserve">
+    <value>¡Destripe!</value>
+  </data>
+  <data name="TrendingSubreddits.Text" xml:space="preserve">
+    <value>Tendencias en Subreddits</value>
+  </data>
+  <data name="TryItTip.Text" xml:space="preserve">
+    <value>¡Adelante! ¡Vuelve a intentarlo!</value>
+  </data>
+  <data name="ViewCommentTip.Text" xml:space="preserve">
+    <value>Para ver los comentarios para este o cualquier por, pon el puntero del ratón o el dedo en la barra del título y navega hacia abajo.</value>
+  </data>
+  <data name="WelcomeText.Text" xml:space="preserve">
+    <value>Bienvenido Quinbd</value>
+  </data>
+  <data name="AccountAgeTB.Text" xml:space="preserve">
+    <value>0 días de antiguedad</value>
+  </data>
+  <data name="AllTime.Text" xml:space="preserve">
+    <value>Todo</value>
+  </data>
+  <data name="CakeDayTB.Text" xml:space="preserve">
+    <value>¡Día del pastel!</value>
+  </data>
+  <data name="CommentSort.Text" xml:space="preserve">
+    <value>Ordenar:</value>
+  </data>
+  <data name="CommentSortFMIBest.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CommentSortFMIContro.Text" xml:space="preserve">
+    <value>Controvérsico</value>
+  </data>
+  <data name="CommentSortFMINew.Text" xml:space="preserve">
+    <value>Nuevo</value>
+  </data>
+  <data name="CommentSortFMITop.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CommentSortNew.Text" xml:space="preserve">
+    <value>Nuevo</value>
+  </data>
+  <data name="CopyLinkMFITB.Text" xml:space="preserve">
+    <value>Copiar enlace</value>
+  </data>
+  <data name="CopyPermaLinkMFITB.Text" xml:space="preserve">
+    <value>Copiar enlace permanente</value>
+  </data>
+  <data name="FormattedPreviewTB.Text" xml:space="preserve">
+    <value>Previsualización Formateada</value>
```

Comment:
I'm not sure about it. I can understand the meaning of this, but I don't know if many people would.
I'll give you here the benefit of the doubt.